python/tests/orderby_tests/sqlite_runner.py

```diff
@@ -0,0 +1,89 @@
+import sys
+import sqlite3
+from types import ModuleType
+import re
+
+from tests.aggregate_tests.aggtst_base import Table, View, TstAccumulator, DEBUG
+from tests.aggregate_tests.atest_register_tests import register_tests_in_module
+
+
+class SQLiteRunner:
+    """Run Tables and Views on SQLite using a sqlite3 cursor"""
+
+    def __init__(self, con: sqlite3.Connection):
+        self.con = con
+        self.cur = con.cursor()
+
+    def run_table(self, table: Table):
```

Comment:
Let's call this function "fill_table"
The description (using """) should say that this inserts the data from table into the sqlite table with the same name